test/manager.cpp

```diff
@@ -0,0 +1,27 @@
+#include "gtest/gtest.h"
+#include "../include/manager.h"
+#include "../include/library.h"
+#include "../include/book.h"
+#include <iostream>
+#include <fstream>
+
+TEST(ManagerTest, addBookFromPathAndGetIdTest)
+{
+    kiwix::Library lib;
+    kiwix::Manager manager = kiwix::Manager(&lib);
+
+    auto bookId = manager.addBookFromPathAndGetId("./test/example.zim");
+    EXPECT_NE(bookId, "");
+    kiwix::Book book = lib.getBookById(bookId);
+    EXPECT_NE(book.getPath(), "");
+    
+    const std::string pathToSave = "./pathToSave";
+    const std::string url = "url";
+    bookId = manager.addBookFromPathAndGetId("./test/example.zim", pathToSave, url, true);
+    book = lib.getBookById(bookId);
+    auto savedPath = isRelativePath(pathToSave) ? 
```

Comment:
You already know that `pathToSave` is relative.
And by the way, you should not use the same code that the code you want to test to compute the right result.